### DISCUSSION TITLE
Plugin Suggestions: Chugging Barrel Preset Names

### DISCUSSION THREAD

**Discussion by davjet1** [runelite/runelite#18411] [Ideas]
Could someone make a chugging barrel preset names plugin? Rn its unlabeled and kinda hard to use. It might be functionality jagex is planning to add but I think a plugin wouldn't hurt in the case they don't do this.

**Comment by evilgenious448** [comment]
@DapperMickie made the Rune pouch loadouts plugin. Maybe if you loan him a chugging barrel, he would be open to making it, but I know he has no interest in grinding for one himself, or spending GP on it.

**Comment by Syhlex** [comment]
I might look into this.

**Reply by evilgenious448** [reply]
I encourage you to use the Rune Pouch Loadouts plugin as a foundation for how you approach this. Keeping the UI uniform between the two would be good for users =]

**Reply by Syhlex** [reply]
Created the plugin: https://github.com/runelite/plugin-hub/pull/6806
